addons/binding/org.openhab.binding.elerotransmitterstick/src/main/java/org/openhab/binding/elerotransmitterstick/internal/EleroTransmitterStickHandlerFactory.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.elerotransmitterstick.internal;
+
+import static org.openhab.binding.elerotransmitterstick.internal.EleroTransmitterStickBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.config.discovery.DiscoveryService;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
+import org.openhab.binding.elerotransmitterstick.internal.discovery.EleroChannelDiscoveryService;
+import org.openhab.binding.elerotransmitterstick.internal.handler.EleroChannelHandler;
+import org.openhab.binding.elerotransmitterstick.internal.handler.EleroTransmitterStickHandler;
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * The {@link EleroTransmitterStickHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "discovery.elerotransmitterstick")
```

Comment:
Remove the `configurationPid` (as it is the wrong one and you do not have any configuration anyhow).

Reply:
Done.